addons/web_editor/static/src/js/editor/rte.js

```diff
@@ -332,9 +332,23 @@ var RTEWidget = Widget.extend({
         });
 
         // start element observation
+        let pastingData = false;
+        $(document).on('paste', function (ev) {
+            pastingData = [...self.editable()];
+            setTimeout(function(){ pastingData = false; }, 0);
```

Comment:
```suggestion
            browser.setTimeout(function(){ pastingData = false; }, 0);
```
I think in 14.0 `browser.setTimeout` is preferred (it is `window.setTimeout.bind(window)` so currently the same, but I think this allow the call to be overriden in a test)